logo-ann/generation/02_generate_embeddings.py

```diff
@@ -1,28 +1,53 @@
 import argparse
+from email.policy import default
 import pathlib
 from typing import Iterable, Set
 
 from efficientnet_pytorch import EfficientNet
+from transformers import CLIPModel, CLIPProcessor
 import h5py
 from more_itertools import chunked
 import numpy as np
 import tqdm
 import torch
+import PIL
 
 from utils import get_offset, get_seen_set
 
+"""
+Returns a hdf5 file containing the embeddings of every logo of the input hdf5 file.
```

Comment:
Same comment as above about PEP-8 formatting.
And you should use (still following PEP-8 rules) use infinitive verbs in docstring (Returns -> Return)